frontend/pubspec.yaml

```diff
@@ -61,6 +63,7 @@ flutter:
   uses-material-design: true
 
   assets:
+    - assets/
     - assets/category_icons/
     - assets/detail_headers/
```

Comment:
We probably do not need the subfolders any more now, right?

Reply:
Subdirectories are not included automatically so we have to create an entry for every subdirectory

Reply:
Oh, okay, good to know 👍🏽